frontend/src/routes/login/Login.svelte

```diff
@@ -92,14 +92,46 @@
         font-size: 20px;
         color: rgb(103, 20, 247)
     }
+
+    h1{
+        margin: auto;
+        text-align: center;
+    }
+
+    #modalForm{
+        display: inline-block;
+        width: 88%;
+        height: 80%;
+        margin-left: 38px;
+    }
+
+    #newLogin, #newPassword, #newPassword2{
+        box-sizing: border-box;
+        width: 90%;
+        height: 40px;
+        border-radius: 5px;
+        padding-left: 10xp;
+        font-size: 16px;
+        margin-top: 10px;
+    }
+
+    #create{
+        width: 180px;
+        margin-left: 70px;
+    }
 </style>
 
 <Modal {opened} id="modal" target={"body"} on:close={() => (opened = false)}>
-    <h1>Create new account</h1>
-    
+    <h1>Create new account</h1><br>
```

Comment:
`<br>` isn't good practice at all.

Here's why:
https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br